rp/tests/manage_content.rs

```diff
@@ -172,41 +173,32 @@ fn should_list_exclusive_content_multiple_items() {
3c1aU0xNE9YWmVpNE1DU0dOaGJtbHpkR1Z5Z3dHREFZSUVXQ0N5WUpjNmV1dVZDS2VnX2hIWU8zY2t6R3poMDhySXg3eXRNemZHelpZQVdvTUJnZ1JZSUJHWTgzUVZYSnNzOGU2bEJxRUlSMVlPRWF6NldQXzZSQUhTb1JyVkVrU1Bnd0dDQkZnZ3lUa21yb2JpbFpWYWh2WERCRWZwUllFS1FsUWdjLUJFdE1LaEFtSlduakNEQVlNQmd3R0NCRmdnU21mcWgzOFJ2ejI0ei0zVTd3TUhiaDU5NmYydFlqQUgwM0pKR3BKSlFzMkRBWUlFV0NDbFpuUXNPNnBzZWRnMjJDYVQxT21KUGtUN1JVOHowUGVvaTRzOFJFWnBSSU1CZ3dHREFZSUVXQ0RKd1BhdlBVWk41UDNaalR2dUI0ck93d2Q1VzhtajloREVFcnJTVWt3RWxZTUJnd0dDQkZnZ05qWVFKdmNfNUdEUGhKd3RpVEFwS0xZUUNaX09TZXpCbHdodWxwOUZua0NEQVlJRVdDRHE4MHZZb0FmOUxhT1ZZMTVFOHJfMndybzVpZERvLUc0WEsxanQyVGJRaklNQmdnUllJSGo2RFdlVFg3N1A3MG5JaWl6LVlsUXoxTHZBRE5WeVBBdHZYT1hFU2RpVGd3SktBQUFBQUFIZ09rRUJBWU1CZ3dHREFrNWpaWEowYVdacFpXUmZaR0YwWVlJRFdDQ2JIUV9aejV5bGJZSEpoN3R2Sl9HaFZubm9DMzdYNHVISHJkOFZtQ2hfOVlJRVdDQ3VTemxvaFVqYTBkci1PNW10M1RWMVRROUZXWExsV0lVWnFzUUVPZ0ZmRVlJRVdDQUpicVVvaUROUzBtbnpMZzY4OGlTejgzelJZTzRCYnFzLTgxbHhPRVhkXzRJRVdDQ3p5U1IwYXdRYkw2OUJQSERvbjRQLW94OXlDckdUZG92N19nR1FTaERCQ0lJRVdDRFgzS0p3YmdYOXNuRDNZT1hHZ0plVERUSFRBakJIMEM4MzByelZ4LUQydG9JRVdDQUtkUnk5RzQ1aGxMaUpKcnFmVHNzejJvcXFIc24xaFdUVUlpVzBLMnN4NDRJRVdDQUxKZFNVNWZPNVE0dlI5cE5wUDVVSEhfV1N1TlBlVDlhWlpuSUV0X2tKR0lJRVdDQWhWWGJ2QXJkZTBDSzFJbEVrM3F3UmNLQm1wUWJOdUFPMUFnQ1VoVEJWeDRJRVdDQkl6SFlqVl9uV2ZId0lNNWE5WFZ4SFpyc0lZbGVucXBuMWkxQU1IQmNacm9JRVdDRGVNUGNrYzFKRWNnZkVFbExEMEFSb1M3WFktbjRnazI5ZDJjMUo3YkI1Q0lNQmdnUllJTEs3VzgxR1Bub0paNGFEc3NWS0lZNVo2LV9WUUlFYVd4dG5IUzVHLTFUWGd3SkVkR2x0WllJRFNiWHk1OHVxNk1HQ0dHbHphV2R1WVhSMWNtVllNS2hBZ0ZldVgxdVF5QkdQX01XRG1RbDRsZVY2QnMydmxNZlJSTnpkbmdTaHNHMlU4bWRIMW9jemRxTTFUN3lIZG1wa1pXeGxaMkYwYVc5dW9tbHpkV0p1WlhSZmFXUllIWk5scU1ueWYwRXdZSVVJTmRHRm1rRmNaMWRjMHd3RW9ZUDlDV01DYTJObGNuUnBabWxqWVhSbFdRSjkyZG4zb21SMGNtVmxnd0dDQkZnZ3ZRNEhOYnN4bkdYTkwwdVhvSUl4ajdFdWdxV21GUGNOSVc5a3BpNkRsdnlEQVlNQmdnUllJTGxKQTJ5d09RbHFuTGYzMzFqcmdFUlJ6VmJBS1FVVkRaTkpGNjB3ZmRoN2d3SkdjM1ZpYm1WMGd3R0RBWUlFV0NDNFM3ZHdZa3JCZEllbjVpSGRjUmJ3LUF6WXZCTWtDOVk2eVBSbVAxX3lVSU1CZ3dHQ0JGZ2daVThpS2cxM2hjbEFhMHpEQ1hEcTVLeE9tU0dScWlDWEFGb3hXVWlzNmR1REFZSUVXQ0RJQ2FMM0hFZzJlZW9JUDhvOXBZTjk0cEJ0NEhNVm5IWnJRNVktbnU3cWFJTUJnZ1JZSUpibzIzeWFLREp1NndlMkJmdTRZSVJZdGc2bUJXcU9GLU1YeGdYMGdPd3Bnd0pZSFpObHFNbnlmMEV3WUlVSU5kR0Zta0ZjWjFkYzB3d0VvWVA5Q1dNQ2d3R0RBazlqWVc1cGMzUmxjbDl5WVc1blpYT0NBMWdiMmRuM2dZSktBQUFBQUFIZ0FBQUJBVW9BQUFBQUFlX19fd0VCZ3dKS2NIVmliR2xqWDJ0bGVZSURXSVV3Z1lJd0hRWU5Ld1lCQkFHQzNId0ZBd0VDQVFZTUt3WUJCQUdDM0h3RkF3SUJBMkVBaXBHeU93bUs2MzNWNVNWeGZ3d3AwNVo4WDA1aDBMWEh4cGNYTG05V2dxcGVRajFDQjNhVnhSZXJwLU8yTnRxVUNUalgxY3J1NG13OGNjeG9pUFZDeFpadVlPcE5rZDI1V3RDWlFvTVhFSmZqSlFKcjd3QXhLTFJHUVFWc2h0QktnZ1JZSU5scEt6TzVOaUhmcGJaQlN1Y1NyUWlDb1BZcGRzTjhVT3JjcFY3MlpvbVFnZ1JZSVBOZURBbnZvcFZsZlpzekFSVF81UlYxdFN0RnNhRFJ1T3c1UmMwTlFnRTFnd0pFZEdsdFpZSURTWkRodnEzbGtPT0JHR2x6YVdkdVlYUjFjbVZZTUpqdHMtMkVKaE5NMTBfc2ZhU3hDQjg0Vm1UWWM1X0FGeWZaYmNwNUVvOUVCOHdHbmlrNW1wVXh2U0NQdnpieXoyUjBjbVZsZ3dKRGMybG5nd0pZSUd1YVFhaWIxMmFnNUxvNWdCOEZLTFlBdGhFNTY3alNWN1B6V0xFdklBc05nd0pZSUQ4WF9xUEtyaUowZ2pXdXFXZUFoZmF6aWhWWFUzcXZVVWhDZE9tNkpZTW9nZ05BIl19fQ.".to_string(),
```

Comment:
Hmm, why is this presentation much longer than the one previously used? 🤔 
I've looked inside the jws via jwt.io, and it seems that the signature-portion of the JWT-encoded VCs is much longer now -- has something changed in the signature encoding?  (I see also that now id-alias VC has an additional field `derivationOrigin`, but that doesn't cause such an increase in the size)

Reply:
Yes, indeed. It's the signature that makes it grow so much. This is also from mainnet. Could it be the difference between mainnet and local replica?

You can also see the same pattern in the other dummy JWS

Reply:
I see, probably the hash tree and/or delegation in the certificate/signature is larger.